src/www/services_unbound_advanced.php

```diff
@@ -350,7 +354,24 @@
                         </td>
                       </tr>
                       <tr>
-                        <td></td>
+                          <td><a id="help_for_extended_statistics" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext('Extended statistics') ?></td>
+                          <td>
+                              <input name="extended_statistics" type="checkbox" id="extended_statistics" value="yes" <?= empty($pconfig['extended_statistics']) ? '' : 'checked="checked"' ?> />
+                              <div class="hidden" data-for="help_for_extended_statistics">
+                                  <?= gettext("If  enabled, extended  statistics are printed. Default is off, because keeping track of more statistics takes time.") ?>
+                              </div>
+                          </td>
+                      </tr>
+                      <tr>
+                          <td><a id="help_for_log_queries" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?= gettext('Log Queries') ?></td>
+                          <td>
+                              <input name="log_queries" type="checkbox" id="log_queries" value="yes" <?= empty($pconfig['log_queries']) ? '' : 'checked="checked"' ?> />
+                              <div class="hidden" data-for="help_for_log_queries">
+                                  <?= gettext("Prints one line per query to the log, with the log timestamp and IP address, name, type and class.  Default is no.  Note that  it takes time to print these lines which makes the server (significantly) slower.  Odd  (nonprintable)  characters  in  names  are printed as '?'.") ?>
+                              </div>
+                          </td>
+                      </tr>
+                      <tr><td></td>
```

Comment:
```suggestion
                      <tr>
                        <td></td>
```

this should clear up the final diff

Reply:
Yes, not part of changed content but nice to clean up anyway. :) well spotted